deploy/azuredeploy.json

```diff
@@ -603,7 +603,10 @@
                         "osType": "Linux",
                         "storageProfile": "ManagedDisks",
                         "vnetSubnetID": "[variables('kubernetesSubnetId')]",
-                        "dnsPrefix": "agents"
+                        "dnsPrefix": "agents",
+                        "maxPods": 30,
+                        "type": "VirtualMachineScaleSets",
```

Comment:
Are the remaining params: `dnsPrefix/maxPods/enableAutoScaling` manadatory ?

Reply:
maxPods should be required when "enableAutoScaling" is true

Reply:
if not, lets get rid of them ?

Reply:
done